app/src/newdocs/content/SDKs.mdx

```diff
@@ -12,7 +12,18 @@ import { JavascriptExample } from './code/api.js'
 <ScrollableAnchor id="sdks-overview"><div>
 
 ## Overview
---content--
+There are both official and unofficial SDKs (Software Development Kits) available for building software that interacts
+with the Streamr platform. Using an SDK provides an effortless and reliable way to use Streamr from your software
+projects. An SDK hides the intricacies involved of interacting with Streamr behind an intuitive library API, allowing a
+developer to focus on their task at hand.
+
+
+Currently, we provide official SDKs for Javascript (browser and Node.js) and Java. There is also an experimental SDK for
+Python in the works. The range of features each of these SDKs provide may vary; be sure to check the documentation of
+a specific SDK for more information.
+
+If your programming language or software environment of choice does not have an SDK available, you can still interact
+with Streamr using our [JSON-based HTTP API](api).
```

Comment:
```suggestion
with Streamr using our <Link to={links.newdocs.api}>JSON-based HTTP API</Link> .
```

Reply:
this style of `<Link>` is better for internal redirects because it uses React Router (no page refresh).